tests/pages/integration.html

```diff
@@ -32,15 +32,24 @@
 
   document.write(collector_endpoint)
 
+  var eventMethod = parseQuery().eventMethod;
+
   window.snowplow('newTracker', 'cf', collector_endpoint, {
```

Comment:
nit - cf is short for cloudfront. Makes no odds really but maybe a different name is preferable?

Reply:
I've removed all the `cf` references in v3.0.0 when I deprecated all the "old" features. I'll ensure these new tests have any `cf` references removed in v3 too, I left them in this release for consistency with the other tests 🤷